src/server/api/medias/videores.js

```diff
@@ -34,4 +33,11 @@ router.get('/subjectareas', (req, res) => {
   );
 });
 
+router.get('/terms', (req, res) => {
+  if (!CheckRoleServices.isAdmin(res.locals.token.roles)) {
+    return res.status(403).send(new Error('Unauthorized role'));
+  }
+  VideoresServices.getTerms().then(terms => res.send(terms));
+});
+
```

Comment:
Seems like the 3 /terms are doing the same thing. Is it possible to move to /api/medias/index.js route?